libraries/Iridium/IdentityManager.test.ts

```
@@ -0,0 +1,60 @@
+import IdentityManager from './IdentityManager'
+const crypto = require('crypto')
+
+Object.defineProperty(global.self, 'crypto', {
+  value: {
+    getRandomValues: (arr) => crypto.randomBytes(arr.length),
+  },
+})
+
+describe('', () => {
+  test('generateEntropyMessage', () => {
```

I might have structured these tests with a describe for each function, i.e.:

describe someFn:
  it should throw an error if a string is passed in
  it should return a uint8array
  it should execute prescribed some computation

The verbosity here helps us to be semantic about what we are testing and why, and makes it clear from a quick review which features we are promising or supporting.

---

btw I also added additional jest matchers [here](https://github.com/Satellite-im/Core-PWA/pull/4300) that will be available when the PR gets merged that could be useful when creating tests